assets/css/styles.css

```diff
@@ -308,7 +308,7 @@ p {
 }
 
 img.landing-logo {
-  margin-bottom: 20px;
+  margin-bottom: 10px;
```

Comment:
margin is still different on mobile. Left margin is also set with a translateX somewhere else, these have yet to be changed.

HackDavis logo should remain fully within "Hack" on desktop. Removing the padding also makes the logo too large on mobile. It's new height is now the height of the text and buttons combined, ie. it takes up half the size of that div. 

I would rather increase the size of the logo on desktop and tweak the padding so it looks the same.